genrl/core/actor_critic.py

```diff
@@ -76,7 +82,20 @@ def __init__(
         self.state_dim = state_dim
         self.action_dim = action_dim
 
+    def get_params(self):
+        actor_params = list(self.shared.parameters()) + list(self.actor.parameters())
+        critic_params = list(self.shared.parameters()) + list(self.critic.parameters())
+        return actor_params, critic_params
+
     def get_features(self, state: torch.Tensor):
+        """Extract features from the state, which is then an input to get_action and get_value
+
+        Args:
+            state (:obj:`torch.Tensor`): The state(s) being passed
+
+        Returns:
+            features (:obj:`torch.Tensor`): The feature(s) extracted from the state
+        """
         features = self.shared(state)
         return features
```

Comment:
Ideally, we should add the option of shared layers to the normal MLP's later on. Let's keep it like this for now because right now everything is stable and tested well. Think of a way we can do this for the future though. Once you merge this, let's track it in a separate issue

Reply:
I could add something like:
```
def get_action(self, state):
    if shared_layers is not None:
        state = self.shared(state)
    action = self.actor(state)
```
and similarly for the critic as well.